api/src/docker/openscad/runScad.js

```diff
@@ -10,13 +10,14 @@ module.exports.runScad = async ({
     camera: {
       position = { x: 40, y: 40, z: 40 },
       rotation = { x: 55, y: 0, z: 25 },
+      dist = 200,
     } = {},
   } = {}, // TODO add view settings
 } = {}) => {
   const tempFile = await makeFile(file)
   const { x: rx, y: ry, z: rz } = rotation
   const { x: px, y: py, z: pz } = position
-  const cameraArg = `--camera=${px},${py},${pz},${rx},${ry},${rz},300`
+  const cameraArg = `--camera=${px},${py},${pz},${rx},${ry},${rz},${dist}`
```

Comment:
Turns out this dist value is important, I don't exactly know why openscad needs this since it can be dirived from the other values, ie dist is just `sqrt(px^2, py^2, pz^2)`

Oh maybe it's expecting a unit vector for px-z? but it also fine with larger values . . . I don't know 🤷